cliquet/session/redis.py

```diff
@@ -43,7 +45,10 @@ def delete(self, key):
 
 def load_from_config(config):
     settings = config.registry.settings
-    host = settings.get('session_redis.host', 'localhost')
-    port = settings.get('session_redis.port', 6379)
-    db = settings.get('session_redis.db', 1)
-    return RedisSessionStorage(host=host, port=port, db=db)
+    uri = settings.get('cliquet.session_url', '')
+    uri = urlparse.urlparse(uri)
+
+    return RedisSessionStorage(host=uri.hostname or 'localhost',
```

Comment:
Having it here allows to have different default values depending on the backend we use. For instance, the port of MySQL can differ from the port of Redis, so it's rather hard to put the defaults in the `.get()` call.